### DISCUSSION TITLE
Copilot for Teachers

### DISCUSSION THREAD

**Discussion by aenichols** [community/community#29974] [Copilot Conversations] [answered]
Will Copilot ever be free for teachers?

**Comment by iamgoodbytes** [comment]
The answer I have received until now is: no. But your students can!

**Comment by maxwellamaral** [comment, marked as answer]
Now you can https://github.blog/2022-09-08-github-copilot-now-available-for-teachers/

**Comment by maxwellamaral** [comment]
Some professors from my educational institution were able to access Copilot, but I couldn't, even though I was registered as a professor on Github and being also linked to my college. How to go about fixing this?

![image](https://user-images.githubusercontent.com/107955034/190410869-65b3bd81-c0f4-4207-bebd-4d1697ff530d.png)


![image](https://user-images.githubusercontent.com/107955034/190411015-2b921e8c-c24e-4b12-b997-c2d79de70880.png)

I proceeded with a new registration. Will this solve the problem?

![image](https://user-images.githubusercontent.com/107955034/190411170-1dcad746-4b79-40e1-958b-f8a1411d9f57.png)

**Reply by samus-aran** [reply]
Hi @maxwellamaral. If you are already a verified teacher then you should have access. I'm not sure if having had the technical preview may have affected your ability to sign up for the free for Teachers Copilot. 

Have a look at your Settings > CoPilot page on your GitHub account and see if you have the option to enable. This [comment](https://github.com/community/community/discussions/19837#discussioncomment-3599754) has helped others.
